main.go

```diff
@@ -20,11 +20,11 @@ import (
 
 const (
 	DefaultMultiCoreNum = 4
+	Path                = "./Log/"
```

Comment:
This const should be defined in Log module, as it is a part of log function, main function don't need care about this